arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -881,6 +884,8 @@
 			dstPath = "";
 			dstSubfolderSpec = 7;
 			files = (
+				0094D34724E769D5007671A3 /* ConfigureSubnetworkTraceViewController.swift in CopyFiles */,
+				0094D34524E76907007671A3 /* ConfigureSubnetworkTraceOptionsViewController.swift in CopyFiles */,
```

Comment:
These two files have the wrong id, so when the project file is saved, Xcode removes them. You'll need to re-add these files to the Copy Files build phase.